kythe/cxx/verifier/BUILD

```diff
@@ -129,13 +129,14 @@ cc_library(
     alwayslink = True,
 )
 
-cc_binary(
-    name = "souffle_with_kythe",
-    deps = [
-        ":souffle_entrystream_support",
-        "@souffle//:main",
-    ],
-)
+# TODO(zarko): Re-enable this target once we can build with c++17 on RBP.
```

Comment:
typo: "RBE"?

Also would be good to be specific here. We do build with C++17, but that doesn't include `<filesystem>` for whatever reason.